scripts/src/main/resources/scripts/command/build

```diff
@@ -50,12 +50,12 @@ then
   fi
 else
   echo "Currently not in a buildable project or build system could not be auto-detected"
-  exit -1
+  exit 255
 fi
 
 if [ -n "${1}" ]
 then
-  BUILD_OPTS="${@}"
+  BUILD_OPTS="${*}"
```

Comment:
I do not know the difference between those two, but I assume you changed this on purpose and consider it as an improvement so if tests work out I am fine with it.

Reply:
I should have added the links to related shellsheck error codes into the bode of the commit messages, sorry. This one was [SC2124](https://github.com/koalaman/shellcheck/wiki/SC2124),

The use of $@ here is an implicit cast - the result by using $* does not change here. (It would be different, if IFS were changed - but this is not the case). See [Special parameters and shell variables](https://wiki.bash-hackers.org/syntax/shellvars#special_parameters_and_shell_variables)

Nevertheless, when I reread the change within whole file context now, my conclusion was wrong: It is the BUILD_OPTS var, which need to be an array, no string. At least that were the right use, when space containing options should be handled correctly. A very rare but not impossible case. Imagine e.g. a logfile path parameter in cygwin bash, where some path element contains a space.

We should skip the change here, since that keeps the shellcheck warning alive, something is here not yet perfect. 

By the way: There is a vscode-shellcheck extension